Dockerfile

```diff
@@ -5,9 +5,10 @@ RUN apt-get install -y --no-install-recommends npm
 COPY . /app
 
 RUN npm install --production
-RUN ./bin/peep.py install --no-cache-dir -r requirements/dev.txt
-RUN ./bin/peep.py install --no-cache-dir -r requirements/prod.txt
-RUN ./bin/peep.py install --no-cache-dir -r requirements/docker.txt
+RUN ./bin/pipstrap.py
```

Comment:
I don't see why this would be necessary, given that pipstrap was run in the base dockerfile.